src/h3/stream.rs

```diff
@@ -132,6 +132,9 @@ pub struct Stream {
 
     /// Whether the stream has been initialized.
     initialized: bool,
+
+    /// Whether the stream has been locally initialized.
+    local_initialized: bool,
```

Comment:
A little confused on the naming -- initialized vs local_initialized. Probably a better clarifying name such as 'header_sent' or something like that?

Reply:
since h3::Stream handles all stream types, having a `headers_sent` message would also seem weird.

I could rename `initialized` to `remote_intialized`...

Reply:
Ok. In the diff `local_initialized` seems only used for checking HEADERS frame sent earlier so I thought `headers_sent' (or `is_headers_sent`) may be better naming. If you think it can be used for other purpose it will be fine.

Reply:
Yeah, renaming `initialized` seems like a good idea, it's kind of confusing right now. Or maybe call them `initial_frame_sent` / `initial_frame_received`? It seems that's what they are used to check for and they are a bit more descriptive.

Reply:
Not all streams take a frame. For example, Push streams start with a Push ID, we just don't have to code implemented to make that clear right now. 

I think local/remote is the appropriate path forward here.